models/balancer/ethereum/balancer_ethereum_trades.sql

```diff
@@ -0,0 +1,115 @@
+{{ config(
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "balancer",
+                                \'["bizzyvinci"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2020-03-13' %}
+
+with v1 as (
```

Comment:
please follow the standard set for splitting spells up across project versions in spellbook. a good example is [uniswap](https://github.com/duneanalytics/spellbook/tree/main/models/uniswap/ethereum), a model per version, with each rolling up into a parent model -- similar to `balancer_trades.sql` file, but for versions on one blockchain, rather than blockchains rolling up.

i'm commenting here on ethereum, but worthwhile to check other blockchains in the PR if multiple versions are deployed on the blockchain